test/integration/functional_test.go

```diff
@@ -158,7 +158,7 @@ func validateDockerEnv(ctx context.Context, t *testing.T, profile string) {
 		t.Fatalf("failed to do minikube status after eval-ing docker-env %s", err)
 	}
 	if !strings.Contains(rr.Output(), "Running") {
-		t.Fatalf("expected status output to include 'Running' after eval docker-env but got: *%q*", rr.Output())
+		t.Fatalf("expected status output to include 'Running' after eval docker-env but got: *%s*", rr.Output())
```

Comment:
- Go tests should have "got before want": https://github.com/golang/go/wiki/TestComments#got-before-want
- The asterisks no longer make sense. I think they are a remnant from when they encapsulated the "want" string.

Reply:
In this case the got is a full output of the stdout and stderr.
So it will be like
Got ...
....
...
...
(Tens of lines if output)
...
Want xyz

It's will be harder to read it. What do you think ?

Thank
Want -- -- but got ( tens of lines of output)

Reply:
Makes sense. Just take the asterisks out. They were there to represent that it's a string submatch.